packages/oscal-react-library/src/components/OSCALControlProse.js

```diff
@@ -25,9 +25,7 @@ const OSCALStatementEditControlsContainer = styled(Grid)`
   text-align: right;
 `;
 
-const NotImplementedStatement = styled(Typography)(
-  ({ theme }) => `color: ${theme.palette.grey[400]};`
-);
+const NotImplementedStatement = styled(Stack)(({ theme }) => `color: ${theme.palette.grey[400]};`);
```

Comment:
This should probably not be a `Stack`... In fact, whatever is using this should probably just use the component you created in #790 right?